webpack/assets/javascripts/hosts/tableCheckboxes.js

```diff
@@ -194,7 +211,7 @@ function isMultple() {
 
 function getBulkParam() {
   return isMultple()
-    ? { search: $('#search').val() }
+    ? { search: getURIsearch() || '' }
```

Comment:
should we move the empty string response to the function ?

Reply:
yes, moved :)